unabridged-works-volume-2/original/03-new-old-covenant.adoc

```diff
@@ -30,17 +30,15 @@ By _Isaac Penington,_ the Younger
 [.section-date]
 1660
 
-[quote.section-epigraph]
+[quote.section-epigraph, , Heb. 4:3]
```

Comment:
I wasn't sure if I should have changed these, but they looked a little funny in the PDF and style guide said to do them this way, but maybe you wanted them the original way? Lines 38 and 1574 as well.